web/src/upload.js

```diff
@@ -426,13 +426,13 @@ export function initialize() {
         mode: "compose",
     });
 
-    // Allow the main panel to receive drag/drop events.
-    $(".app-main").on("dragover", (event) => event.preventDefault());
+    // Allow the app panel to receive drag/drop events.
+    $(".app").on("dragover", (event) => event.preventDefault());
 
     // TODO: Do something visual to hint that drag/drop will work.
-    $(".app-main").on("dragenter", (event) => event.preventDefault());
+    $(".app").on("dragenter", (event) => event.preventDefault());
 
-    $(".app-main").on("drop", (event) => {
+    $(".app").on("drop", (event) => {
```

Comment:
Would `$(".app, #navbar-fixed-container")` work as a selector to cover the navbar too?

I guess the other option is to do `body`, which would add overlays to what is covered, but I don't think we want drag/drop onto the settings overlay to work (and might conflict with dragging custom emoji, for example)

Reply:
I definitely prefer the idea of using a multi-element selector more. But the `#navbar-fixed-container` will also cover the navbar notifications as well, do we want to include that?
![localhost_9991_](https://github.com/zulip/zulip/assets/82862779/3252ffb6-908e-411a-a848-661fe9c4a3ba)

Reply:
Also, it came to my notice the `.app` selector contains the **About Zulip** and some hidden form related to logout. We can fix this by wrapping the `app-main` container inside a wrapper and then targeting it, how does that sound?

Reply:
*  I doubt anyone will ever drag an image onto those green banners, but if they do, they likely are trying to drop it into Zulip, not replace the Zulip window that image. 
* The hidden logout form is invisible, so not important. But I also expect it could be happily moved to its proper place in the DOM; I think it's part of the gear menu "Logout" button.
* It's probably a bug that the "About Zulip" modal gets appended to `.app` rather than `body` or a dedicated block like `user-profile-modal-holder`? I doubt anyone will drag anything onto it, so it's certainly not worth creating an extra div over it, but it may be worth giving it an `about-zulip-modal-container` div right after `user-profile-modal-holder` while we're paying attention to this.

Reply:
Address all the feedbacks. One thing that I would like to bring to your notice is that when I tried to place the logout form in the navbar it led to the csrf_token token issue and the browser window would endlessly reload. So I instead ended up appending it to the body, since that made sense for it to be globally available.